tests/providers/google/cloud/operators/test_dataproc.py

```diff
@@ -1064,11 +1064,10 @@ def test_create_cluster_operator_extra_links(dag_maker, create_task_instance_of_
     serialized_dag = dag_maker.get_serialized_data()
     deserialized_dag = SerializedDAG.from_dict(serialized_dag)
     deserialized_task = deserialized_dag.task_dict[TASK_ID]
-
     # Assert operator links for serialized DAG
-    assert serialized_dag["dag"]["tasks"][0]["__var"]["_operator_extra_links"] == [
-        {"airflow.providers.google.cloud.links.dataproc.DataprocClusterLink": {}}
-    ]
+    deserialized_dag = SerializedDAG.deserialize_dag(serialized_dag["dag"])
```

Comment:
@Taragolis -> this is how I corrected the tests that were somewhat relying on internals of serialization.